README.md

```diff
@@ -27,4 +36,12 @@ You need to create a new profile (or edit an existing one) on the Minecraft
 launcher and modify the profile to look like the above but replace the path
 to steven with the location you built it at or downloaded it too and change the 
 `server` parameter to the target server. Currently only works in online mode
-(with no plans for offline mode currently).
\ No newline at end of file
+(with no plans for offline mode currently).
+
+It is possible to run steven without the launcher, but you must obtain the access token,
+UUID (whithout dashes) and the username, and pass them as arguments to steven, as well as
+the server.
+
+## Controls
+
+Currently it is not possible to chat or interact with anything, but `w` will move you in the direction your cursor is pointing.
```

Comment:
I'd prefer to leave the controls out of the readme for now, they'll be changing soon